spec/lib/pdf_filler/nj1040_pdf_spec.rb

```diff
@@ -641,6 +641,8 @@
         }
 
         it "includes the sum 200,000 split into each box on line 15" do
+          submission.data_source.synchronize_df_w2s_to_database
```

Comment:
i think so, but open to other approaches? i landed on this because it was easier to call the synchronize method than to manually create the w2 records based on the fixture contents. i think it would also be acceptable to create state file w2s from the factory

Reply:
Could this be in a before each for every test in this file?

Reply:
@aloverso i was also thinking that i didn't love calling this method in a bunch of places so impulsively did an experiment at the end of yesterday: https://github.com/codeforamerica/vita-min/pull/4886/commits/a504b987e3f7351a777607c0e6a4db1d7291390b

let me know what you think!